server/controller/articleController.js

```diff
@@ -80,6 +81,109 @@ class ArticleController {
       })
     }
   }
+
+  static async updateArticle(req, res) {
+    try {
+      const { articleId } = req.params;
+      const { title, article } = req.body;
+      const user = await pool.query(queries.selectArticle,[articleId]);
+
+        if (user.rowCount === 0) return res.status(404).json({ 
+            status: 'Failed',
+            message: 'Article Not Found'
+        });
+        
+        const updatedAt = DateTime.now()
+        const updated = await pool.query(queries.updateArticle,[title, article, updatedAt, articleId]);
```

Comment:
for this you will always be updating the title and article, although its not bad but its always better to do a patch that is only those that are changing. so if only the title is changing update that alone. well it means you have to tweak this a bit. but don't worry lets keep it this way. just wanted to mention this here